### ISSUE TITLE
VSCODE unable to run flutter apps after recent update

### ISSUE THREAD

**Issue by Puspharaj** [microsoft/vscode#100736]
- VSCode Version:Version: 1.46.1 (user setup)
Commit: cd9ea6488829f560dc949a8b2fb789f3cdc05f5d
Date: 2020-06-17T21:13:20.174Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

Steps to Reproduce:

1.Click on run flutter. This ends up in cannot delete folders.
2.

Does this issue occur when all extensions are disabled?: Yes/No --- no 

```

What went wrong:
Execution failed for task ':app:cleanMergeDebugAssets'.
> Unable to delete directory 'D:\AllWorksUbuntu\Projects\flutterSample\build\app\intermediates\merged_assets\debug\out'
    Failed to delete some children. This might happen because a process has files open or has its working directory set in the target directory.
    - D:\AllWorksUbuntu\Projects\flutterSample\build\app\intermediates\merged_assets\debug\out\flutter_assets\packages
    - D:\AllWorksUbuntu\Projects\flutterSample\build\app\intermediates\merged_assets\debug\out\flutter_assets

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 52s
Exception: Gradle task assembleDebug failed with exit code 1
```

**Comment by Puspharaj** [comment]
I tried to move the entire code from D drive to C drive and no luck....this is something to do with vscode folder permission issue i guess. My entire work had been lost today because of this stupid bug.

`FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:generateDebugBuildConfig'.
> java.nio.file.AccessDeniedException: C:\Projects\engine_sense\build\app\generated\source\buildConfig\debug\in\engine_sense

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 9s
Exception: Gradle task assembleDebug failed with exit code 1
Exited (sigterm)`

**Comment by eamodio** [comment]
Please open an issue with the flutter extension.

**Comment by Phearun99** [comment]
Fixed in VS Code (version 1.47.0-insider)
